packages/e2e-tests/helpers/pause-information-panel.ts

```diff
@@ -45,6 +45,18 @@ export async function expandAllScopesBlocks(page: Page): Promise<void> {
   });
 }
 
+export async function ensureSidePanelClosed(page: Page) {
+  // Clicks that aren't directly on an element can cause the "Comments" pane to open.
```

Comment:
I don't understand this comment. What "elements"? Why would clicking on something randomly open the pause panel?

Looking at the places you're calling this (after `warpToMessage` and after `verifyStackingTestCaseSelectedElementUnderCursor` I'm kind of concerned that one of our test helpers is really flaky and we should **fix it** rather than this bandaid.

Reply:
Ah, that's interesting. Yeah now there's a context menu that you have to explicitly click "add comment" in.

Maybe we can remove this and see if things still pass?

Reply:
Yeah, seems to pass if I just call it once at the start of the test. (I still want it closed in some cases to increase the size of the video/elements pane.)